examples/mnist/README.md

```diff
@@ -14,4 +14,4 @@ I0527 09:46:50.714237 139989059204928 mnist_lib.py:178] eval epoch: 10, loss: 0.
 
 ### How to run
 
-`python mnist_main.py --model_dir=/tmp/mnist`
```

Comment:
Can we make there be a default value for `--config` so that it's not passed in explicitly every time?

Reply:
+1

Reply:
Passing a default value for --config is definitely possible. But since the config paths work with relative paths based on the current directory of execution, I don't think we can fix a reasonable default value. 

i.e if someone runs the mnist example from mnist/ dir, the path will be `configs/default.py`. Let's say the Main is executed from flax/, the config path becomes `examples/mnist/configs/default.py`.

Reply:
What about `os.path.join(os.path.dirname(__file__), 'configs/default.py'))` ?

Reply:
Thanks for the suggestion. Done!